src/kbmod/run_search.py

```diff
@@ -20,12 +20,12 @@ class run_search:
 
     Parameters
     ----------
-    input_parameters : ``dict``
-        Input parameters. Merged with the ``defaults`` dictionary.
-        Must contain `im_filepath` and ``res_filepath`` keys, paths to
-        the image and results directory respectively. Should contain
-        ``v_arr``, and ``ang_arr``, which are lists containing the lower
-        and upper velocity and angle limits.
+        input_parameters : ``dict``
+            Input parameters. Merged with the ``defaults`` dictionary.
+            Must contain `im_filepath` and ``res_filepath`` keys, paths to
+            the image and results directory respectively. Should contain
+            ``v_arr``, and ``ang_arr``, which are lists containing the lower
+            and upper velocity and angle limits.
```

Comment:
This doesn't need an indent. Whatever warning or error message you saw, it wasn't because of a lack of indent. Making it all indented makes is a quote-block and then likely doesn't parse whatever caused the error.

Reply:
Ok. I am sort of guess and check here so probably did it poorly. Could you contrast the two pages (first is installed, 2nd is my latest build):

https://epyc.astro.washington.edu/~kbmod/api_reference/run_search_referenceapi.html#kbmod.run_search.run_search.run_search

https://epyc.astro.washington.edu/~cchris28/api_reference/run_search_referenceapi.html#kbmod.run_search.run_search.run_search